next/test/fields.test.ts

```diff
@@ -24,11 +24,11 @@ describe('fields', () => {
     ])
   })
 
-  it('should throw an error if the type is an array', () => {
+  it('should throw an error if the type equals "array"', () => {
```

Comment:
todo: we can be more explicit on what it means:

```suggestion
  it('should throw an error if the type equals "array" (group-array)', () => {
```